BookMyShowApp.API/Interface/ITheatres.cs

```diff
@@ -0,0 +1,19 @@
+﻿using BookMyShow.Data;
+using BookMyShow.Data.ViewModels;
+
+namespace BookMyShowApp.API.Interface
+{
+    public interface ITheatres
+    {
+       Task<IEnumerable<Theatre>> GetTheatres(int CityId);
```

Comment:
1. Format code properly in all files
2. Remove unnecessary new lines from all files.
3. Fix naming conventions for parameter names